### ISSUE TITLE
Please, update to 1.21 and 1.21.1

### ISSUE THREAD

**Issue by KM-Owner** [OneLiteFeatherNET/AntiRedstoneClock-Remastered#90]
### Server Logs

```
[14:17:17] [Server thread/INFO]: [AntiRedstoneClock-Remastered] Enabling AntiRedstoneClock-Remastered v1.2.0
[14:17:17] [Server thread/INFO]: [AntiRedstoneClock-Remastered] To keep the project alive, we ask for donations to our https://opencollective.com/onelitefeather/projects/antiredstoneclock-remastered
[14:17:18] [Server thread/ERROR]: Error occurred while enabling AntiRedstoneClock-Remastered v1.2.0 (Is it up to date?)
java.lang.NullPointerException: Cannot invoke "com.github.zafarkhaja.semver.Version.isHigherThan(com.github.zafarkhaja.semver.Version)" because "this.remoteVersion" is null
	at AntiRedstoneClock-Remastered-1.2.0.jar/net.onelitefeather.antiredstoneclockremastered.service.UpdateService.notifyConsole(UpdateService.java:47) ~[AntiRedstoneClock-Remastered-1.2.0.jar:?]
	at AntiRedstoneClock-Remastered-1.2.0.jar/net.onelitefeather.antiredstoneclockremastered.AntiRedstoneClockRemastered.updateService(AntiRedstoneClockRemastered.java:115) ~[AntiRedstoneClock-Remastered-1.2.0.jar:?]
	at AntiRedstoneClock-Remastered-1.2.0.jar/net.onelitefeather.antiredstoneclockremastered.AntiRedstoneClockRemastered.onEnable(AntiRedstoneClockRemastered.java:102) ~[AntiRedstoneClock-Remastered-1.2.0.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:288) ~[paper-mojangapi-1.21-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:202) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:109) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:520) ~[paper-mojangapi-1.21-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.CraftServer.enablePlugin(CraftServer.java:654) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at org.bukkit.craftbukkit.CraftServer.enablePlugins(CraftServer.java:603) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:756) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:518) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:363) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1229) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:333) ~[purpur-1.21.jar:1.21-2275-82ccc76]
	at java.base/java.lang.Thread.run(Thread.java:1570) ~[?:?]
[14:17:18] [Server thread/INFO]: [AntiRedstoneClock-Remastered] Disabling AntiRedstoneClock-Remastered v1.2.0
```

### Expected behavior

Please, update to 1.21 and 1.21.1

### Actual behavior

-

### Steps to reproduce

-

### Plugin and Datapack List

-

### Paper version

purpur-1.21.jar:1.21-2275-82ccc76

### Plugin version

1.2.0

**Comment by TheMeinerLP** [comment]
Is already fixed, Update to 1.2.1-SNAPSHOT -> https://hangar.papermc.io/OneLiteFeather/AntiRedstoneClock-Remastered/versions/1.2.1-SNAPSHOT+51

**Comment by KM-Owner** [comment]
Thank you!
